src/main/java/com/socrata/api/Soda2Producer.java

```diff
@@ -76,6 +78,7 @@ public static final Soda2Producer newProducerWithRequestId(final String url, Str
     public Soda2Producer(HttpLowLevel httpLowLevel)
     {
         super(httpLowLevel);
+        factory = new JsonFactory();
```

Comment:
@gilday What email address do you use for your Socrata developer account? I can get grant your account access to create datasets so you can run the tests.

We should also replace those with environment variables so I can finish the work of setting up continuous integration...